main/src/ca/uwaterloo/flix/language/phase/sjvm/BytecodeCompiler.scala

```diff
@@ -53,9 +140,145 @@ object BytecodeCompiler {
 
   def pushInt[R <: Stack](n: Int): F[R] => F[R ** PrimInt32] = ???
 
+  def pushChar[R <: Stack](c: Char): F[R] => F[R ** PrimChar] = ???
+
+  object ArrayInstructions {
```

Comment:
I think we should put *all* instructions inside an object `Instruction.scala` and then do a wildcard import so we can use them like `DUP`, `BALoad` etc.